### PR TITLE
Centaur Warunner Buffs

### PR DESCRIPTION
**Pull request by POCKETMONEY99**
Set hoofstomp cooldown to 13 seconds at all levels. This should help improve centaurs early game contest etc. 

Reduced Mana Cost on Hoofstomp at OAA levels slightly.

Increased Return Damage at OAA levls

### CODE REVIEW

File: game/scripts/npc/abilities/centaur_double_edge.txt [hunk 1]
```diff
@@ -39,7 +39,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "strength_damage"                                 "60 80 100 120 140 180"
+        "strength_damage"                                 "60 80 100 120 180 240"
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/centaur_double_edge.txt]: This ability on Centaur is the only good scaling ability on Centaur. This is also a percentage of his strength. Not cool. Please revert.

File: game/scripts/npc/abilities/centaur_hoof_stomp.txt [hunk 2]
```diff
@@ -35,7 +35,7 @@
       "02"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "stun_duration"                                   "1.75 2.0 2.25 2.5 2.65 2.75" //OAA
+        "stun_duration"                                   "2.0 2.25 2.5 2.75 2.75 2.75"
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/centaur_hoof_stomp.txt]: Talent value is back to normal. It was +1s, now its +1.5s. I think that change was a good buff already. 4.25s AoE stun in mid game is not cool. In late game you have enough status resistance so its fine. Scaling it from 2.0 at lvl 1 to 2.75 at lvl 6 would be ideal: 2/2.15/2.3/2.45/2.6/2.75. It buffs its early game, not cancer in mid game, and normal at lategame. But I also think current scaling (1.75/2./2.25/2.5/2.65/2.75) is fine as well. :man_shrugging:

Comment (POCKETMONEY99) [reply game/scripts/npc/abilities/centaur_hoof_stomp.txt]: ok

File: game/scripts/npc/abilities/centaur_double_edge.txt (hunk 1 above)
Comment (POCKETMONEY99) [reply game/scripts/npc/abilities/centaur_double_edge.txt]: ok

**Pull request merged**